endpoints/getting-started-grpc/README.md

```diff
@@ -0,0 +1,178 @@
+# Endpoints Getting Started with gRPC & Python Quickstart
+
+It is assumed that you have a working Python environment and a Google
+Cloud account and [SDK](https://cloud.google.com/sdk/) configured.
+
+1. Install dependencies using virtualenv:
+
+    ```bash
+    virtualenv -p python3 env
+    source env/bin/activate
+    pip install -r requirements.txt
+    ```
+
+1. Test running the code, optional:
+
+    ```bash
+    # Run the server:
+    python greeter_server.py
+
+    # Open another command line tab and enter the virtual environment:
+    source env/bin/activate
+
+    # In the new command line tab, run the client:
+    python greeter_client.py
+    ```
+
+1. The gRPC Services have already been generated. If you change the proto, or
+   just wish to regenerate these files, run:
+
+    ```bash
+    python -m grpc_tools.protoc -I protos --python_out=. --grpc_python_out=. protos/helloworld.proto
+    ```
+
+1. Generate the `out.pb` from the proto file:
+
+    ```bash
+    python -m grpc_tools.protoc --include_imports --include_source_info -I protos protos/helloworld.proto --descriptor_set_out out.pb
```

Comment:
It's fine, I was just questioning Jeff, but he's on leave now so it's whatever. :)